pkgs/development/python-modules/starlette/default.nix

```
@@ -20,12 +20,18 @@
 
 buildPythonPackage rec {
   pname = "starlette";
-  version = "0.13.0";
+
+  # This is not the latest version of Starlette, however, later
+  # versions of Starlette break FastAPI due to
+  # https://github.com/tiangolo/fastapi/issues/683. Please update when
+  # possible. FastAPI is currently Starlette's only dependent.
+
+  version = "0.12.9";
   disabled = isPy27;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "6bd414152d40d000ccbf6aa40ed89718b40868366a0f69fb83034f416303acef";
+  src = fetchurl {
+    url = "https://github.com/encode/starlette/archive/${version}.tar.gz";
```

please use fetchFromGitHub helper
```suggestion
  src = fetchFromGitHub {
    owner = "encode";
    repo = pname;
    rev = version;
```

otherwise LGTM

---

Done now. Should be ready to merge.